.ci/gcb-generate-diffs.yml

```diff
@@ -110,41 +110,78 @@ steps:
           - $_PR_NUMBER
 
     - name: 'gcr.io/graphite-docker-images/downstream-builder'
-      id: tfv-head
       secretEnv: ["GITHUB_TOKEN"]
-      waitFor: ["merged", "tpg-head"]
+      waitFor: ["merged"]
       args:
           - 'head'
-          - 'terraform-validator'
-          - 'ga'
+          - 'tf-oics'
+          - 'beta'
           - $_PR_NUMBER
 
     - name: 'gcr.io/graphite-docker-images/downstream-builder'
-      id: tfv-base
       secretEnv: ["GITHUB_TOKEN"]
-      waitFor: ["merged", "tpg-base"]
+      waitFor: ["merged"]
       args:
           - 'base'
-          - 'terraform-validator'
+          - 'tf-oics'
+          - 'beta'
+          - $_PR_NUMBER
+
+    - name: 'gcr.io/graphite-docker-images/terraform-tester'
+      id: tpgb-test
+      secretEnv: ["GITHUB_TOKEN"]
+      waitFor: ["tpgb-head", "tpgb-base"]
+      args:
+          - 'beta'
+          - $_PR_NUMBER
+          - $COMMIT_SHA
+          - $BUILD_ID
+          - $PROJECT_ID
+          - GoogleCloudPlatform/magic-modules
+          - "15"  # Build step
+
+    - name: 'gcr.io/graphite-docker-images/terraform-tester'
+      id: tpg-test
+      secretEnv: ["GITHUB_TOKEN"]
+      waitFor: ["tpg-head", "tpg-base"]
+      args:
           - 'ga'
           - $_PR_NUMBER
+          - $COMMIT_SHA
+          - $BUILD_ID
+          - $PROJECT_ID
+          - GoogleCloudPlatform/magic-modules
+          - "16"  # Build step
+
+    - name: 'gcr.io/graphite-docker-images/gcb-terraform-vcr-tester'
```

Comment:
This step needs to happen after `diff`. Error:

```
invalid build: invalid .steps field: build step https://github.com/GoogleCloudPlatform/magic-modules/pull/17 - "gcb-tpg-vcr-test" depends on "diff", which has not been defined
```

https://github.com/GoogleCloudPlatform/magic-modules/pull/7517/checks?check_run_id=12205682779

Reply:
Done